lessons/basic_stats/lesson.md

```diff
@@ -10,3 +10,330 @@ visible: true
+migraine$gender <- c("F", "F", "F", "M", "M", "M", "M", "F", "F", "M", "M", "F", "F", "F", "M", "F", "M", "M", "M", "F", "F", "F", "F", "M", "F", "M", "M")
+migraine
+```
+
+```
+##    pain drug gender
+## 1     4    A      F
+## 2     5    A      F
+## 3     4    A      F
+## 4     3    A      M
+## 5     2    A      M
+## 6     4    A      M
+## 7     3    A      M
+## 8     4    A      F
+## 9     4    A      F
+## 10    6    B      M
+## 11    8    B      M
+## 12    4    B      F
+## 13    5    B      F
+## 14    4    B      F
+## 15    6    B      M
+## 16    5    B      F
+## 17    8    B      M
+## 18    6    B      M
+## 19    6    C      M
+## 20    7    C      F
+## 21    6    C      F
+## 22    6    C      F
+## 23    7    C      F
+## 24    5    C      M
+## 25    6    C      F
+## 26    5    C      M
+## 27    5    C      M
+```
+
+```r
+#Run two-way ANOVA using gender and drug as factors
+results = aov(pain ~ drug*gender, data=migraine)
+summary(results)
+```
+
+```
+##             Df Sum Sq Mean Sq F value   Pr(>F)    
+## drug         2 28.222  14.111  28.088 1.16e-06 ***
+## gender       1  0.002   0.002   0.004    0.952    
+## drug:gender  2 17.893   8.946  17.808 3.00e-05 ***
+## Residuals   21 10.550   0.502                     
+## ---
+## Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1
+```
+
+```r
+#Post hoc analysis on two-way ANOVA
+TukeyHSD(results, conf.level = 0.95)
+```
+
+```
+##   Tukey multiple comparisons of means
+##     95% family-wise confidence level
+## 
+## Fit: aov(formula = pain ~ drug * gender, data = migraine)
+## 
+## $drug
+##          diff       lwr       upr     p adj
+## B-A 2.1111111  1.268923 2.9532993 0.0000084
+## C-A 2.2222222  1.380034 3.0644104 0.0000040
+## C-B 0.1111111 -0.731077 0.9532993 0.9410333
+## 
+## $gender
+##            diff        lwr       upr     p adj
+## M-F -0.01648352 -0.5842182 0.5512512 0.9524246
+## 
+## $`drug:gender`
+##          diff        lwr         upr     p adj
+## B:F-A:F  0.30 -1.1875023  1.78750227 0.9873049
+## C:F-A:F  2.20  0.7975694  3.60243059 0.0009237
+## A:M-A:F -1.20 -2.6875023  0.28750227 0.1619424
+## B:M-A:F  2.60  1.1975694  4.00243059 0.0001207
+## C:M-A:F  1.05 -0.4375023  2.53750227 0.2752703
+## C:F-B:F  1.90  0.4124977  3.38750227 0.0074874
+## A:M-B:F -1.50 -3.0679651  0.06796506 0.0660199
+## B:M-B:F  2.30  0.8124977  3.78750227 0.0010862
+## C:M-B:F  0.75 -0.8179651  2.31796506 0.6700989
+## A:M-C:F -3.40 -4.8875023 -1.91249773 0.0000064
+## B:M-C:F  0.40 -1.0024306  1.80243059 0.9441652
+## C:M-C:F -1.15 -2.6375023  0.33750227 0.1947322
+## B:M-A:M  3.80  2.3124977  5.28750227 0.0000011
+## C:M-A:M  2.25  0.6820349  3.81796506 0.0024216
+## C:M-B:M -1.55 -3.0375023 -0.06249773 0.0379545
+```
+
+```r
+#Plot the interaction for our two-way ANOVA
+interaction.plot(drug, gender, pain)
+```
+
+![center]({{ site.github.url }}/img/lessons/basic_stats/lesson/unnamed-chunk-1-1.png)
```

Comment:
Unfortunately, Jekyll doesn't like that. Maybe Jekyll's updated this, but last time I tried doing that it as too complicated since Jekyll creates the md file into a `filename/index.html` format... thus moving the file up one and breaking relative paths. Unless I make the relative files `../`.. But it's easy to add this line using `knitr` so. What do you think?